corebehrt/functional/creators.py

```diff
@@ -9,26 +9,26 @@
                               get_time_difference, normalize_segments_df)
 
 
-def create_ages(concepts: pd.DataFrame, birthdates: dict) -> pd.DataFrame:
+def create_ages(concepts: pd.DataFrame, patients_info: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I think it's more expressive to have the required inputs for the function in `functional`, rather than passing in `concepts` and `patients_info`

I think we should have `def create_ages(timestamp: pd.Series, birthdates: pd.Series)` and handle the other things outside the `functional` library (perhaps inside `classes`)